package.json

```diff
@@ -31,9 +31,9 @@
   },
   "homepage": "https://github.com/digicatapult/dscp-ipfs#readme",
   "dependencies": {
-    "@polkadot/api": "^5.9.1",
+    "@digicatapult/dscp-node": "latest",
```

Comment:
Don't use latest here, lets at least assume we can maintain some sort of semver and use `^3.0.1`

Reply:
changed